config/agents/svc/rbac/kustomization.yaml

```diff
@@ -6,3 +6,4 @@ resources:
 - service_agent_manager_role_binding.yaml
 - service_agent_manager_role.yaml
 - service_agent_account.yaml
+namespace: services
```

Comment:
Since `system` is used everywhere else, shouldn't we be using that as the default for `kustomize`?

When we do deployments in acceptance testing, we can always override it either by changing this namespace with `kustomize edit` or by overriding it in `kubectl`.

Reply:
nice catch, this change was not supposed to be commited